src/AppKit/NSPrintInfo.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using XamCore.PrintCore;
+
+namespace XamCore.AppKit {
+	public partial class NSPrintInfo {
+		public PMPrintSession PMPrintSession {
```

Comment:
Any reason for not using `PrintSession` ? i.e. drop the `PM` prefix. The other properties of `NSPrintInfo` don't seems to be prefixed and if they were they would be wrong ;-)